src/util/fs.js

```diff
@@ -454,3 +455,33 @@ export async function getFileSizeOnDisk(loc: string): Promise<number> {
 export function normalizeOS(body: string): string {
   return body.replace(/\r\n/g, '\n');
 }
+
+const cr = new Buffer('\r', 'utf8')[0];
+const lf = new Buffer('\n', 'utf8')[0];
+
+async function getEolFromFile(path: string) : Promise<string | void>  {
+  let buffer;
+  try {
+    buffer = await readFileBuffer(path);
```

Comment:
better return undefined if files does not exist (check if it exists first).
And don't catch error in readFile